doc/cephfs/eviction.rst

```diff
@@ -102,7 +102,7 @@ the blacklist:
 
     ceph osd blacklist ls
```

Comment:
Include the output of `blacklist ls` and make sure the address matches what is passed to `blacklist rm`.

Reply:
Done